src/bionics.cpp

```diff
@@ -370,7 +511,7 @@ bool Character::activate_bionic( int b, bool eff_only )
 
         // We can actually activate now, do activation-y things
         mod_power_level( -bio.info().power_activate );
-        if( bio.info().toggled || bio.info().charge_time > 0 ) {
+        if( bio.info().has_flag( "BIONIC_TOGGLED" ) || bio.info().charge_time > 0 ) {
             bio.powered = true;
         }
```

Comment:
You can replace things like this with:
```suggestion
        bio.powered = bio.info().has_flag( "BIONIC_TOGGLED" ) || bio.info().charge_time > 0;
```